### DISCUSSION TITLE
USR-W610 - Has anyone had any success with this adapter?

### DISCUSSION THREAD

**Discussion by akgp195** [kellerza/sunsynk#97] [General]
This seems a little cheaper and more available in the UK that the USR-630. 

Has there been anyone who's had this working before I pull the trigger on one?

@kellerza et al...
Firstly thanks for this as I was almost going to go the solor-assistant route !
Secondly the differences listed in the page below makes me think this will work. Do you think it's worth a try?

https://www.pusr.com/news/differences-among-serial-to-wifi-converter-products.html

All the best 
Albert

**Comment by kellerza** [comment]
It needs to support modbus rtu to tcp (or modbus tcp)

Good luck & if it works, please add the the docs!

**Reply by kellerza** [reply]
Did you manage to get this working?
